exporter/clickhousemetricsexporter/clickhouse.go

```diff
@@ -359,12 +341,9 @@ func (ch *clickHouse) Write(ctx context.Context, data *prompb.WriteRequest, metr
 		return err
 	}
 
-	for k, v := range metrics {
-		stats.RecordWithTags(ctx, []tag.Mutator{tag.Upsert(usage.TagTenantKey, k), tag.Upsert(usage.TagExporterIdKey, ch.exporterID.String())}, ExporterSigNozSentMetricPoints.M(int64(v.Count)), ExporterSigNozSentMetricPointsBytes.M(int64(v.Size)))
-	}
-
 	// write to distributed_samples_v4 table
 	if ch.writeTSToV4 {
```

Comment:
this will always be true ?

Reply:
Yes https://github.com/SigNoz/signoz-otel-collector/blob/140c30d2c73e5925c6460107382d85c90b0edda6/exporter/clickhousemetricsexporter/factory.go#L128